charts/ceph-csi-rbd/values.yaml

```diff
@@ -0,0 +1,215 @@
+---
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: true
+
+serviceAccounts:
+  nodeplugin:
+    # Specifies whether a ServiceAccount should be created
+    create: true
+    # The name of the ServiceAccount to use.
+    # If not set and create is true, a name is generated using the fullname
+    name:
+  provisioner:
+    # Specifies whether a ServiceAccount should be created
+    create: true
+    # The name of the ServiceAccount to use.
+    # If not set and create is true, a name is generated using the fullname
+    name:
+
+socketDir: /var/lib/kubelet/plugins/cephfs.csi.ceph.com
+socketFile: csi.sock
+registrationDir: /var/lib/kubelet/plugins_registry
+pluginDir: /var/lib/kubelet/plugins
+driverName: cephfs.csi.ceph.com
```

Comment:
same for driver name

Reply:
😳 copy paste errors, sorry about that. Fixed all references now :)

Reply:
still, I see the issue
```
[🎩︎]mrajanna@localhost ceph-csi-rbd $]grep cephfs -ri *
values.yaml:socketDir: /var/lib/kubelet/plugins/cephfs.csi.ceph.com
values.yaml:driverName: cephfs.csi.ceph.com
[🎩︎]mrajanna@localhost ceph-csi-rbd $]pwd
/home/mrajanna/workspace/src/github.com/ceph/ceph-csi/charts/ceph-csi-rbd
[🎩︎]mrajanna@localhost ceph-csi-rbd $]

```

```
[🎩︎]mrajanna@localhost ceph-csi-cephfs $]grep rbd -ri *
Chart.yaml:  - https://github.com/ceph/ceph-csi/tree/master/charts/ceph-csi-rbd
README.md:kubectl delete namespace ceph-csi-rbd
```

Reply:
Sure you pullled the last commit?

```console
~/Development/Prive/ceph-csi grep cephfs -ri charts/ceph-csi-rbd/*
~/Development/Prive/ceph-csi grep rbd -ri charts/ceph-csi-cephfs/*
```

Reply:
nope its not present now, Thanks